provider/resource_overrides.rb

```diff
@@ -112,8 +112,8 @@ def override_properties(api_object, override)
                              property_override, Provider::PropertyOverride
         api_property = find_property api_object, property_path.split('.')
         if api_property.nil?
-          raise "The property to override must exists #{property_path} " \
-                "in resource #{api_object.name}"
+          raise "The property to override '#{property_path}' must exists in " \
```

Comment:
If you feel like another grammar change, this oughta be 'must exist'.

Reply:
Done